third_party/sycl/crosstool/CROSSTOOL.tpl

```diff
@@ -84,7 +84,6 @@ toolchain {
   unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
 
-  compiler_flag: "-no-serial-memop"
```

Comment:
Are we definitely removing this again? Feels like we've just added it, and it's (fairly) portable.

Reply:
I should have removed it only for cross-compilation. Now it actually makes more sense to have this option in all of our configs except sycl_arm

Reply:
My understanding of -no-serial-memop is that it removes a workaround that we need for some buggy OpenCL drivers (on both desktop and embedded devices). If we have it set by default then we will cause problems for users that follow our guides to build and run TensorFlow on such an OpenCL device, and the errors that they see will not be obviously related to this flag (as far as I know). So I believe that it should be removed here and also from the bazel.rc. Users can specify it in their build if they already verified that their build works without it

Reply:
Ok thanks for the info. We removed it from our scripts and will add the option only on the command line when needed.